sign-and-combine-ecdsa/nodejs/.env.example

```
@@ -0,0 +1,4 @@
+ETHEREUM_PRIVATE_KEY=
+LIT_PKP_PUBLIC_KEY=
+LIT_PKP_ETH_ADDRESS=
+CAPACITY_TOKEN_ID=
```

```suggestion
LIT_CAPACITY_CREDIT_TOKEN_ID=
```
This is what we use in the other examples